assets/js/index.js

```diff
@@ -0,0 +1,39 @@
+const addIcon = document.getElementById("add-icon");
+const input = document.getElementById("input");
+const errorHandling = document.getElementById("error-handling")
+
+async function postTodo() {
+  const url = "http://localhost:3030/todos";
+  
+  try {
+    const response = await fetch(url,{
+      method: "POST",
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({text: input.value})
+    });
+    if (response.status != 200) {
+      errorHandling.innerText = "Something went wrong!";
```

Comment:
This message should show up under the input. Also, the message text should come from backend.

Reply:
Done.